reference/strings/functions/crypt.xml

```diff
@@ -210,18 +210,10 @@
 <![CDATA[
 <?php
 
-/* Sie sollten das vollständige Ergebnis von crypt() als Salt zum
-   Passwort-Vergleich übergeben, um Problemen mit unterschiedlichen
-   Hash-Algorithmen vorzubeugen. (Wie bereits ausgeführt, verwendet
-   ein Standard-DES-Passwort-Hash einen 2-Zeichen-Salt, ein
-   MD5-basierter hingegen nutzt 12 Zeichen. */
-if (crypt($benutzer_eingabe, $gehashtes_passwort) == $gehashtes_passwort) {
-   echo "Passwort stimmt überein!";
-
 $benutzer_eingabe = 'rasmuslerdorf';
 $gehashtes_passwort = '$6$rounds=1000000$NJy4rIPjpOaU$0ACEYGg/aKCY3v8O8AfyiO7CTfZQ8/W231Qfh2tRLmfdvFD6XfHk12u6hMr9cYIA4hnpjLNSTRtUwYr9km9Ij/';
 
-// Prüfen eines vorhandenen crypt()-Hashs auf Kompatibilität mit Nicht-PHP-Software .
+// Prüfen eines vorhandenen crypt()-Hashs für Kompatibilität mit Nicht-PHP-Software.
```

Comment:
According to Duden "auf" should be the correct preposition here.

Reply:
I authored the English version, the suggested change is the intended meaning. The English version attempts to say (roughly paraphrased):

> Wir prüfen einen vorhandenen crypt()-Hash einer Drittsoftware, um die Kompatibilität mit dieser Drittsoftware herzustellen.

The current translation with “auf” sounds to me like:

> Wir prüfen ob der vorhandene crypt()-Hash mit der Drittsoftware kompatibel ist.

which is something entirely different. I'll check with a native speaker of English whether the English version should be rephrased.

Reply:
``für`` is also a valid translation for ``for`` checking Leo: https://dict.leo.org/german-english/for

Reply:
OK, I see. Then why not just use this as the translation
```suggestion
// Prüfen eines vorhandenen crypt()-Hashs, um die Kompatibilität mit Nicht-PHP-Software herzustellen.
```

Reply:
Or
```suggestion
// Dafür sorgen, dass ein vorhandener crypt()-Hash mit Nicht-PHP-Software kompatibel ist.
```

Reply:
We've now updated the English version to be a little clearer. The updated version also translates nicely to German.